fftool

```diff
@@ -1969,6 +1953,172 @@ class system(object):
 
             # f.write('\n')
 
+    def writeopenmm(self):
```

Comment:
This is not elegant.
If the omm.py is immutable then we can supply it as a file in some directory with fftool.
In python the way to write chunks of text is as a multiline string """..."""